tasks/data_loading/src/s3_client_utils.py

```diff
@@ -35,3 +35,29 @@ def labeled_sentences_from_json(sents_json):
     """
     return {sent_id: sent_labels_map for sent_id, sent_labels_map in [*sents_json.values()][0]["sentences"].items()}
 
```

Comment:
I think you forgot to define the function here? Because you wrote the comments and the code within the `labeled_sentences_from_json` method and after a `return` statement